### DISCUSSION TITLE
Using NMRium for process .jdf file from a Jeol NMR

### DISCUSSION THREAD

**Discussion by ajmgiraffe** [NFDI4Chem/nmrxiv#838] [General]
Hi, I am new to this chat, I am trying to process NMR spec from Jeol, so the file type is 'jdf. When I drag and drop, the file is there, but there is no spectra to view, whether the nucleus is on or off - please can someone advise me on how to remedy this issue, or tell me if it can't be remedied? Many thanks, Anthony

**Comment by NRayya** [comment]
Dear @ajmgiraffe 
Sorry for the late reply. We don't usually use discussions for this purpose but you can instead send us emails to info.nmrxiv@uni-jena.de . I hope your issue is solved otherwise please let us know.
